contents/handbook/company/culture.md

```diff
@@ -48,13 +46,13 @@ To accomplish this, we use [asynchronous communication](/handbook/company/commun
 
 Putting things in writing helps us clarify our own ideas, as well as allow others to provide better feedback. It has been key to our development and growth.
 
-## Don't let others fail
+## We don't let others fail
```

Comment:
I feel like this is somewhat at odds with our performance expectations and the fact that we do let people go if they aren't able to perform.

Reply:
Yeah I feel like this heading has nothing to do with the content. Maybe "We give direct feedback early and often" and link to the blog post about this?